docs/sources/auth/google.md

```diff
@@ -19,7 +19,7 @@ To enable the Google OAuth2 you must register your application with Google. Goog
 First, you need to create a Google OAuth Client:
 
 1. Go to https://console.developers.google.com/apis/credentials
-2. Click the 'Create Credentials' button, then click 'OAuth Client ID' in the menu that drops down
+2. Click the 'Create Credentials' button, then click 'OAuth Client ID' in the drop down menu
```

Comment:
```suggestion
2. Click **Create Credentials**, then click **OAuth Client ID** in the drop-down menu
```

Reply:
Per Microsoft Style Guide, most clickable elements in the UI are bold. 

Also, see their guidance on the use of "drop-down." https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/d/drop-down